### ISSUE TITLE
Training error with deep auto encoder

### ISSUE THREAD

**Issue by sinaakhtar** [amzn/amazon-dsstne#158]
Hi, I am running a deep auto encoder using DSSTNE but having issues with the training. The first epoch runs returns a training error of > 0, and the second epoch results in 0.

**Comment by ekandrotA9** [comment]
Actually, I have seen something like this recently while adding Batch Norm to DSSTNE.  I had the learning rate sign backwards, a positive instead of a negative when using the results from cuDNN.  Flipping the sign on the learning then caused it to converge.  Until I did that, I had the results just like yours.

So, I'm guessing that it is similar - that it grows without bounds, caps out, then can not change values any more, so average training error drops to zero (ie because they can not change).  How are you invoking DSSTNE?  Do  you have your own main.cpp, or are you using encoder built from main.cpp in utils?

**Comment by scottlegrand** [comment]
I would turn on verbose mode and watch per-minibatch training error to see when it blows up.

**Comment by spacelover1** [comment]
Hello @scottlegrand,

I would like to know if I can use other Error Functions, such as RMSE in the `config.json` file? I couldn't find anything [here](https://github.com/amzn/amazon-dsstne/blob/master/docs/getting_started/userguide.md) about the Error function.
Also would you please tell me where I can add the verbose mode?
